_docs/extensibility/adding-mappings-source.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: docs
+title: Mappings Source Extensions
+meta_title: Mappings Source Extensions
+description: Adding additional source of loading mappings via extensions.
+---
+
+Additional source to load the stub mappings can be configured by implementing `MappingsSourceExtension`.
+
+```java
+public class DummyMappingsSourceExtension extends MappingsSourceExtension {
+  @Override
+  public String getName() {
+    return null; // Return the name of extension
```

Comment:
People will inevitably copy-paste examples verbatim so please can we change this to a non-null String.